Sources/PT.PM.Common/Nodes/UstVisitor.cs

```diff
@@ -178,6 +178,16 @@ public virtual T Visit(IntLiteral intLiteral)
             return VisitChildren(intLiteral);
         }
 
+        public virtual T Visit(BigIntLiteral bigIntLiteral)
+        {
+            return VisitChildren(bigIntLiteral);
```

Comment:
Sorry, what children?

Reply:
This is a basic visitor method, by default it does nothing for any node. This class is used for inheritance if we are only interested in certain types of nodes and the rest can be skipped.

Reply:
Yep, and `VisitChildren` will return `null` when there are no children, why not just immediately return `null`?

Reply:
It's default stub for `Visit` methods.

Reply:
Why not to change the default to just `return null`?

Reply:
it's not possible to use `return null` because we have Generic class, but i change it to return default, where it possible

Reply:
In this case, processing will be terminated for a node type with undefined `Visitor` right away. I suggest leaving it as it.

Reply:
after painful deliberation, I decided to insist on the endurance of uniformity for all methods)